.github/ISSUE_TEMPLATE.md

```diff
@@ -7,11 +7,11 @@ Thanks for opening an issue! A few things to keep in mind:
 - If you need general advice, join our Slack: http://atom-slack.herokuapp.com
 -->
 
-* Electron version:
+* Output of `electron --version`:
```

Comment:
My concern here is this won't be their actual electron version, it will be the globally installed version rather than their locally installed version.

Maybe `node_modules/.bin/electron --version`

Reply:
Great point. Updating the PR to do this instead.